src/care_dvdms/migrations/0004_dvdmsdrug_dvdmsrecordorder_dvdmsrecorditemorder.py

```
@@ -0,0 +1,75 @@
+import uuid
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('care_dvdms', '0003_dvdmsstore'),
+        ('emr', '0080_alter_activitydefinition_category_and_more'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)